src/graphql/client.js

```diff
@@ -0,0 +1,59 @@
+import { createClient, dedupExchange, cacheExchange, fetchExchange } from 'urql';
+import { authExchange } from '@urql/exchange-auth';
+import { makeOperation } from '@urql/core';
+import { SK_PAT, SK_USER } from '../constants';
+import { readFromLocalStorage } from '../api/chrome';
+
+const getAuth = async ({ authState }) => {
+  if (!authState) {
+      const { pat } = await readFromLocalStorage([SK_PAT, SK_USER])
+
+      if (!pat) {
+          throw Error('PAT not available.') 
+      }
+
+      return { token: pat }
+  }
+
+  return null;
```

Comment:
invert conditional to stick to early return pattern, also don't return nulls

Reply:
Yepp. Based on https://formidable.com/open-source/urql/docs/advanced/authentication/ the **getAuth** was returning null. We know that the return value of this is used in the **addAuthToOperation**() function. To not return null, we can return an object: **{ token: null }** instead. Both options work in the **addAuthToOperation**() function, but your suggestion avoid using nulls.
PR updated.